rskj-core/src/test/java/co/rsk/config/RskSystemPropertiesTest.java

```diff
@@ -95,4 +100,34 @@ public void shouldUseExpectedBloomConfigKeys() {
         assertTrue(expectedConfig.hasPath(configKeyCaptorForHasPath.getValue()));
         assertTrue(expectedConfig.hasPath(configKeyCaptorForGetInt.getValue()));
     }
+
+    @Test
+    public void testRpcModules() {
+        RskSystemProperties rskSystemProperties = new RskSystemProperties(
+                new ConfigLoader(
+                        new CliArgs.Parser<>(
+                                NodeCliOptions.class,
+                                NodeCliFlags.class
+                        ).parse(new String[]{})
+                )
+        );
+        assertFalse(
```

Comment:
can we also check if other modules are enabled? it can be done in a loop I guess

Reply:
yes